test/plugin_functional/test_suites/embedding_visualizations/embed_by_id.js

```diff
@@ -0,0 +1,167 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import expect from 'expect.js';
+
+export default function ({ getService }) {
+  const testSubjects = getService('testSubjects');
+  const find = getService('find');
+  const table = getService('table');
+
+  async function selectVis(id) {
+    await testSubjects.click('visSelect');
+    await find.clickByCssSelector(`option[value="${id}"]`);
+  }
+
+  async function selectParams(id) {
+    await testSubjects.click('embeddingParamsSelect');
+    await find.clickByCssSelector(`option[value="${id}"]`);
+    await testSubjects.waitForDeleted('visLoadingIndicator');
```

Comment:
This methods waits the default timeout, and if the network have high latency, it can be cause a timeout error and a "flaky test" like the following generated configuring `slow 3g` on dev tools:
```
fail: "embedding visualizations embed by id vis on timebased data without date histogram should correctly embed"
         │      Timeout: An operation did not complete before its timeout expired.
```
Our tests assumes that for sure the visLoadingIndicator will be removed after the `whenFirstRenderComplete` will complete, so here it's better to try this for X times at least or increase the default timeout to avoid having network timeout errors.